CONTRIBUTING.md

```diff
@@ -179,7 +150,7 @@ Note: These instructions assume the user is using `mamba` instead of `conda` sin
 
 - Pascal architecture or better
```

Comment:
"Pascal architecture GPU or better"
or
"GPU with Compute Capability of 6.0 or higher"